.gitignore

```diff
@@ -1,21 +1,24 @@
-/application.yml
```

Comment:
No, not intentional, but needed to add some things. I'll take a look

On Fri, Nov 6, 2015 at 1:44 PM, Mark Fisher <notifications@github.com>
wrote:

> In .gitignore
> <https://github.com/springone2gx2015/vehicle-fleet-demo/pull/5#discussion_r44172272>
> :
>
> > @@ -1,21 +1,24 @@
> > -/application.yml
>
> Did you intentionally overwrite .gitignore? There are some entries here
> that should not be deleted. For example, I am now seeing .factorypath
> files.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/springone2gx2015/vehicle-fleet-demo/pull/5/files#r44172272>
> .
>